server/service.go

```diff
@@ -121,10 +122,39 @@ func (m *BoostService) handleRoot(w http.ResponseWriter, req *http.Request) {
 	respondOK(w, nilResponse)
 }
 
+// handleStatus sends calls to the status endpoint of every relay.
+// It returns OK if at least one returned OK, and returns KO otherwise.
 func (m *BoostService) handleStatus(w http.ResponseWriter, req *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusOK)
-	fmt.Fprintf(w, `{}`)
+	var wg sync.WaitGroup
+	var numSuccessRequestsToRelay uint32
+
+	for _, r := range m.relays {
+		wg.Add(1)
+
+		go func(relay RelayEntry) {
+			defer wg.Done()
+
+			m.log.WithField("relay", relay).Debug("Checking relay status")
```

Comment:
```suggestion
			m.log.WithField("relay", relay.Address).Debug("Checking relay status")
```

Reply:
maybe better would be to do something like

```suggestion
			log = log.WithField("relay", relay)
			log.Debug("Checking relay status")
```

and later down on the error logging also use the new log method